test/tap/ci-with-local-dependency.js

```diff
@@ -0,0 +1,83 @@
+var fs = require('graceful-fs')
+var path = require('path')
+
+var mkdirp = require('mkdirp')
+var osenv = require('osenv')
+var rimraf = require('rimraf')
+var test = require('tap').test
+
+var common = require('../common-tap.js')
+
+var pkg = common.pkg
+
+var EXEC_OPTS = { cwd: pkg, stdio: [0, 1, 2] }
+
+var localDependencyJson = {
+  name: 'local-dependency',
+  version: '0.0.0',
+  dependencies: {
+    underscore: '1.5.1'
```

Comment:
Oh, I just realized, this is going to hit the public registry during a test.  I'll switch it to use the mock registry, but for future reference, that's usually not a great idea.  With as many tests as npm has, if it had to look further than `localhost`, it's kind of a nightmare.